incubator/binding-mqtt.spec/src/test/java/io/aklivity/zilla/specs/binding/mqtt/streams/network/ConnectionIT.java

```diff
@@ -401,6 +401,15 @@ public void shouldNotReceivePublishPacketExceedingMaxPacketLimit() throws Except
         k3po.finish();
     }
 
+    @Test
+    @Specification({
+        "${net}/connect.reject.will.message.too.large/client",
+        "${net}/connect.reject.will.message.too.large/server"})
+    public void shouldRejectLargeWillMessage() throws Exception
```

Comment:
This is actually the `connect` packet that is too large, not specifically the `will` message.
Suggest naming `connect.reject.packet.too.large` and `shouldRejectPacketTooLarge()`.